src/plugins/graph/models/graph-model.ts

```diff
@@ -187,9 +190,10 @@ export interface IGraphModelSnapshot extends SnapshotIn<typeof GraphModel> {}
 export function createGraphModel(snap?: IGraphModelSnapshot) {
   return GraphModel.create({
     axes: {
-      bottom: EmptyAxisModel.create({place: "bottom"}),
-      left: EmptyAxisModel.create({place: "left"})
+      bottom: NumericAxisModel.create({place: "bottom", min: -10, max: 11}),
+      left: NumericAxisModel.create({place: "left", min: -11, max: 11})
```

Comment:
Similarly, the default axis type could be determined by an `emptyPlotIsNumeric` property available via `useSettingFromStores()`.